File: src/main/java/com/bot/commands/general/PingCommand.java
package com.bot.commands.general;

import com.bot.commands.GeneralCommand;
import com.jagrosh.jdautilities.command.CommandEvent;

public class PingCommand extends GeneralCommand {

    public PingCommand() {
        this.name = "ping";
        this.guildOnly = false;
        this.help = "Gets the ping from Vinny to discord.";
    }

    @Override
    protected void executeCommand(CommandEvent commandEvent) {

        commandEvent.reply(commandEvent.getJDA().getGatewayPing() + "ms");
    }
}

File: src/main/java/com/bot/commands/general/PingCommand.kt
package com.bot.commands.general

import com.bot.commands.GeneralCommand
import com.jagrosh.jdautilities.command.CommandEvent
import datadog.trace.api.Trace

class PingCommand : GeneralCommand() {
    init {
        this.name = "ping"
        this.guildOnly = false
        this.help = "Gets the ping from Vinny to discord."
    }

    @Trace(operationName = "executeCommand", resourceName = "Ping")
    override fun executeCommand(commandEvent: CommandEvent) {
        commandEvent.channel.sendMessage(commandEvent.jda.gatewayPing.toString() + "ms").complete()
    }
}
